src/pool.rs

```diff
@@ -34,6 +34,14 @@ impl<T: TaskCell + Send> ThreadPool<T> {
         self.remote.spawn(t);
     }
 
+    /// Scale workers of the thread pool, the new thread count should be
+    /// smaller than the max_thread_count.
```

Comment:
Better to describe the meaning of the return value.

Reply:
> 

Agree, will be added

Reply:
Fixed, thanks!